app/pom.xml

```diff
@@ -371,22 +372,6 @@
             </executions>
           </plugin>
 
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-source-plugin</artifactId>
-            <configuration>
-              <skipSource>false</skipSource>
-            </configuration>
-            <executions>
-              <execution>
-                <id>attach-sources</id>
-                <goals>
-                  <goal>jar</goal>
-                </goals>
-              </execution>
-            </executions>
-          </plugin>
-
```

Comment:
Don't we want to release sources?

Reply:
Obviously removed the wrong plugin. These is a duplicate error  as there is 2 maven-source-plugin in the release profile.